x-pack/platform/plugins/shared/fleet/server/routes/agent_policy/handlers.ts

```diff
@@ -196,7 +196,7 @@ export const bulkGetAgentPoliciesHandler: FleetRequestHandler<
         'full query parameter require agent policies read permissions'
       );
     }
-    let items = await agentPolicyService.getByIDs(soClient, ids, {
+    let items = await agentPolicyService.getByIds(soClient, ids, {
```

Comment:
Side-effect of removing the usage of AgentPolicyServiceInterface: interface had `getByIDs` and implementation has `getByIds`. I chose the latter to stay, but it's easy to rename implementation to `getByIDs`. This was mostly done to avoid pinging other teams that might have used the interface method name.